data/nation/liege.ruleset

```diff
@@ -0,0 +1,78 @@
+[nation_liege]
+
+name=_("Liegean")
+plural=_("?plural:Liegeans")
+groups="Modern", "European"
+legend = _("Liege was constituted in ecclesiastic principalty in 985,\
+ when Bishop Notger was granted the control of several cities in current Belgium.\
+ As a federated state of the Holy Roman Empire, it was characterized\
+ with large civil liberties for which liegean struggled strongly for.\
+ It lasted until 1792, when the recently formed Republic of Liege voted\
+ the union with revolutionary France.\
+ Liege played later a major role in the Belgian independance.")
+
+leaders = {
+ "name",                   "sex"
+ "Notger",                 "Male"
+ "Wazon",                  "Male"
+ "Albert de Cuyck",        "Male"
+ "Érard de la Marck",      "Male"
+ "Jean-Nicolas Bassenge",  "Male"
+ "Joseph Bologne",         "Male"
+ "Julienne de Cornillon",  "Female"
+ "Théroigne de Méricourt", "Female"
```

Comment:
You could also put some more recent figures like "Ferdinand Piercot", "Émile Digneffe" or "Gustave Kleyer".

Reply:
Too many males already

Reply:
For the record: 
Julienne de Cornillon : https://en.wikipedia.org/wiki/Juliana_of_Li%C3%A8ge
Théroigne de Méricourt : https://en.wikipedia.org/wiki/Theroigne_de_Mericourt